Lib/fontbakery/profiles/fvar.py

```diff
@@ -490,10 +540,11 @@ def com_adobe_fonts_check_varfont_same_size_instance_records(ttFont):
 )
 def com_adobe_fonts_check_varfont_distinct_instance_records(ttFont, has_name_table):
     """Validates that all of the instance records in a given font have distinct data."""
+
+    check_warned = False
```

Comment:
I prefer to use a variable named `passed` (initialized to `True`) because sometimes policies change and checks end up being downgraded from FAIL to WARN (or the other way around) and then it is easier to not have to also rename such variables.

Reply:
Makes sense. Changed in f0a30c21f1633bc10da1fdf97635d1200b056511